app/index.html

```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" href="favicon.svg">
```

Comment:
This should likely be "/favicon.svg". Otherwise it will not load on nested pages (e.g. when on a blogpost page) as it will be appended to the right. Might even want to add {{rootURL}} in front here and for the manifest below as well.